packages/ssz/src/viewDU/listComposite.ts

```diff
@@ -66,6 +66,43 @@ export class ListCompositeTreeViewDU<
     return this.type.getViewDU(newRootNode) as this;
   }
 
+  /**
+   * Returns a new ListCompositeTreeViewDU instance with the values from `index` to the end of list
+   *
+   * ```ts
+   * const nodes = getChunkNodes()
+   * return listFromChunkNodes(node.clice(index))
+   * ```
+   *
+   * Note: If index === n, returns an empty list of length 0
+   *
+   */
+  sliceFrom(index: number): this {
```

Comment:
I feel less confident on the name of the function. `slice` is very common known term and always returns a shallow copy without changing the object. But this implementation is return the current object with mutation. 

May be we call it `sliceAndMerge` or something similar.

Reply:
this function returns new ViewDU object without mutating the current ViewDU object
I think the name is good and it conforms to the `sliceTo` method above

Reply:
Yea it is essentially the opposite of `sliceTo`, so naming and behaviour are similar to it.